### PR TITLE
Putting back the issue template eaten by gremlins!

### PR DESCRIPTION
**Pull request by radinamatic**
## Summary

*Eat this, you nasty buggers!*

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## System information
+
+*Please specify the Kolibri version you were using and your operating system!*
+
+ - Version: ?
+ - Operating system: ?
+ - Browser: ?
+
+## Traceback or relevant snippet from server.log
```
Comment (rtibbles) [inline .github/ISSUE_TEMPLATE.md]: Modify to:
`## Traceback or relevant snippet from server.log or browser console`

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Steps
+1. To
+1. Reproduce
+
+## Screenshots
+
+*If applicable*
+
+## Real-life consequences (anything community should be aware, for instance how it affects your deployment)
```
Comment (rtibbles) [inline .github/ISSUE_TEMPLATE.md]: Modify to:
`## Real-life consequences (anything community should be aware of, for instance how it affects your deployment)`

**Comment by rtibbles** [comment]
I also note that you have failed to add yourself to the AUTHORS.rst. Please do so.

**Comment by radinamatic** [comment]
Corrections added, @rtibbles! :smiley:

**Pull request merged**